mzLib/mzLib.nuspec

```diff
@@ -2,7 +2,7 @@
 <package>
   <metadata>
     <id>mzLib</id>
-    <version>5.0.538</version>
+    <version>5.0.540</version>
```

Comment:
I think this is supposed to be the version of the package. Should this be `1.0.540`?